2015-11-10 06:09:32 Error: Fatal Error (4): syntax error, unexpected ''empty'' (T_CONSTANT_ENCAPSED_STRING), expecting ')' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:09:33 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ''empty'' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:15:36 Error: Fatal Error (4): syntax error, unexpected 'array' (T_ARRAY) in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 16]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:15:37 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'array' (T_ARRAY)
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 16)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:35:55 Error: Fatal Error (4): syntax error, unexpected ';' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 20]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:35:55 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ';'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 20)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:36:47 Error: Fatal Error (4): syntax error, unexpected ';' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 20]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:36:48 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ';'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 20)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:36:55 Error: Fatal Error (4): syntax error, unexpected 'echo' (T_ECHO) in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 21]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:36:56 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'echo' (T_ECHO)
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 21)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:40:29 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:40:29 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ')'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:41:33 Error: Fatal Error (4): syntax error, unexpected '=', expecting ']' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:41:34 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=', expecting ']'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:42:32 Error: Fatal Error (4): syntax error, unexpected '=', expecting ']' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:42:32 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=', expecting ']'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:42:34 Error: Fatal Error (4): syntax error, unexpected '=', expecting ']' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:42:35 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=', expecting ']'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:42:36 Error: Fatal Error (4): syntax error, unexpected '=', expecting ']' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:42:37 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=', expecting ']'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:43:05 Error: Fatal Error (4): syntax error, unexpected '=', expecting ']' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:43:06 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=', expecting ']'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:43:07 Error: Fatal Error (4): syntax error, unexpected '=', expecting ']' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:43:07 Error: Fatal Error (4): syntax error, unexpected '=', expecting ']' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:43:07 Error: Fatal Error (4): syntax error, unexpected '=', expecting ']' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:43:07 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=', expecting ']'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:43:07 Error: Fatal Error (4): syntax error, unexpected '=', expecting ']' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:43:08 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=', expecting ']'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:43:08 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=', expecting ']'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:43:08 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=', expecting ']'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-10 06:50:06 Error: Fatal Error (4): syntax error, unexpected ';' in [C:\xampp\htdocs\cakephp\src\Template\Pizza\add.ctp, line 23]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-11-10 06:50:07 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ';'
Request URL: /cakephp/pizza/add
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 23)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-12-07 06:45:58 Error: [Cake\Datasource\Exception\InvalidPrimaryKeyException] Record not found in table "articles" with primary key [NULL]
Request URL: /cakephp/articles/view
Stack Trace:
#0 C:\xampp\htdocs\cakephp\src\Controller\ArticlesController.php(37): Cake\ORM\Table->get(NULL, Array)
#1 [internal function]: App\Controller\ArticlesController->view()
#2 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#3 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#4 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#5 C:\xampp\htdocs\cakephp\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#6 {main}


2015-12-07 06:47:26 Error: [Cake\Datasource\Exception\RecordNotFoundException] Record not found in table "articles"
Request URL: /cakephp/articles/view/%7B1%7D
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\ORM\Table.php(1187): Cake\ORM\Query->firstOrFail()
#1 C:\xampp\htdocs\cakephp\src\Controller\ArticlesController.php(37): Cake\ORM\Table->get('{1}', Array)
#2 [internal function]: App\Controller\ArticlesController->view('{1}')
#3 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#6 C:\xampp\htdocs\cakephp\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-12-07 06:50:08 Error: [Cake\Datasource\Exception\InvalidPrimaryKeyException] Record not found in table "articles" with primary key [NULL]
Request URL: /cakephp/articles/view
Stack Trace:
#0 C:\xampp\htdocs\cakephp\src\Controller\ArticlesController.php(37): Cake\ORM\Table->get(NULL, Array)
#1 [internal function]: App\Controller\ArticlesController->view()
#2 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#3 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#4 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#5 C:\xampp\htdocs\cakephp\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#6 {main}


2015-12-07 07:14:00 Error: [Cake\Datasource\Exception\RecordNotFoundException] Record not found in table "articles"
Request URL: /cakephp/articles/view/4
Stack Trace:
#0 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\ORM\Table.php(1187): Cake\ORM\Query->firstOrFail()
#1 C:\xampp\htdocs\cakephp\src\Controller\ArticlesController.php(37): Cake\ORM\Table->get('4', Array)
#2 [internal function]: App\Controller\ArticlesController->view('4')
#3 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#6 C:\xampp\htdocs\cakephp\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-12-07 07:14:21 Error: [Cake\Datasource\Exception\InvalidPrimaryKeyException] Record not found in table "articles" with primary key [NULL]
Request URL: /cakephp/articles/view
Stack Trace:
#0 C:\xampp\htdocs\cakephp\src\Controller\ArticlesController.php(37): Cake\ORM\Table->get(NULL, Array)
#1 [internal function]: App\Controller\ArticlesController->view()
#2 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#3 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#4 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#5 C:\xampp\htdocs\cakephp\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#6 {main}


2015-12-07 07:14:28 Error: [Cake\Datasource\Exception\InvalidPrimaryKeyException] Record not found in table "articles" with primary key [NULL]
Request URL: /cakephp/articles/view/
Stack Trace:
#0 C:\xampp\htdocs\cakephp\src\Controller\ArticlesController.php(37): Cake\ORM\Table->get(NULL, Array)
#1 [internal function]: App\Controller\ArticlesController->view()
#2 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#3 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#4 C:\xampp\htdocs\cakephp\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#5 C:\xampp\htdocs\cakephp\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#6 {main}


